testgrid/specs/deploy.yaml

```diff
@@ -64,7 +64,7 @@
 - name: k8s123-minimal
   installerSpec:
     kubernetes:
-      version: 1.23.x
+      version: 1.24.x
```

Comment:
We still want to keep this at `1.23.x` per the test name

Reply:
This was a nice catch! I completely forgot to rename the test. As per our last meeting I have opted to rename the test instead than duplicating it.